.goreleaser.yml

```diff
@@ -6,6 +6,7 @@ builds:
     main: main.go
     # Set the binary output location to bin/ so archive will comply with Sensu Go Asset structure
     binary: bin/{{ .ProjectName }}
+    name_template: "{{ .ProjectName }}-{{ .Tag }}"
```

Comment:
This line appears to be invalid at this location, goreleaser threw me this error:

```
line 9: field name_template not found in type config.Build
```

Reply:
Update: I believe we actually need to set `name_template` in the `archives` section:

```
archives:
  - id: tar
    format: tar.gz
    name_template: "{{ .ProjectName }}_{{ .Tag }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
[...]
```